native/mod_manager/domain.c

```diff
@@ -212,21 +221,35 @@ apr_status_t find_domain(mem_t *s, domaininfo_t **domain, const char *route, con
  * get the ids for the used (not free) domains in the table
  * @param pointer to the shared table.
  * @param ids array of int to store the used id (must be big enough).
- * @return number of domain existing or -1 if error.
+ * @return number of domain existing or 0.
  */
+static apr_status_t loc_get_id(void *mem, void *data, apr_pool_t *pool)
+{
+    struct counter *count = (struct counter *)data;
+    domaininfo_t *ou = (domaininfo_t *)mem;
+    *count->values = ou->id;
+    count->values++;
+    count->count++;
+    return APR_SUCCESS;
+}
 int get_ids_used_domain(mem_t *s, int *ids)
```

Comment:
The same issue as above, the `loc_get_id` should be above the docs for `get_ids_used_domain`.

Reply:
let's keep that like it is: we are planing to replace the loc_get_id()  by something more generic.